### ISSUE TITLE
Need help with "Failed to run: exec: not started"

### ISSUE THREAD

**Issue by zodo** [wagoodman/bashful#35]
I can not run the simplest example.
YML file content:
```
tasks:
    - cmd: echo "Hello, World!"
```
Result of `bashful run hello.yml`:
```
Running hello.yml
    • echo "Hello, World!"      Exited with error (-1)                                                                                                                                    
      100.00% Complete  Tasks[1/1]                                                                                                                                       Runtime[00:00:00]
 ...Some tasks failed, see below for details.

⏺ Failed task: echo "Hello, World!"
  ├─ command: echo "Hello, World!"
  ├─ return code: -1
  ╰─ stderr: Failed to run: exec: not started
```
`go version` is `go version go1.8 linux/amd64`
`go env` is
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/username/.go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build246215807=/tmp/go-build -gno-record-gcc-switches"
CXX="g++"
CGO_ENABLED="1"
PKG_CONFIG="pkg-config"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
```

Looking forward for your support

**Comment by wagoodman** [comment]
I've been taking a look and have not been able to reproduce this issue (same go version and env flags). It seems that the "os/exec" Command has not started before Wait() is invoked (from https://github.com/golang/go/blob/master/src/os/exec/exec.go#L454). 

I'm not certain how this is possible (from https://github.com/wagoodman/bashful/blob/master/task.go#L457 and https://github.com/wagoodman/bashful/blob/master/task.go#L521)

What OS and version are you running on? If you run some of the run yamls from the `examples/` dir do you see the same behavior? 

I'll be revamping the releases such that you can download the latest Linux version from the releases tab... once I get that up and going would you be willing to try out the same hello world example with that precompiled binary?

**Comment by zodo** [comment]
> What distro and version are you running on?

Ubuntu 16.04

>  If you run some of the run yamls from the examples/ dir do you see the same behavior?

Yes, the same error for all yamls

> would you be willing to try out the same hello world example with that precompiled binary?

It would be great. Thank you

**Comment by wagoodman** [comment]
I ran it on a ubunu 16.04 box for kicks, same result (worked fine for me). I just got the releases up, give this a shot https://github.com/wagoodman/bashful/releases/download/v0.0.10/bashful_0.0.10_linux_amd64.tar.gz .

**Comment by zodo** [comment]
Finally, I realized what the problem is.
My `SHELL` env variable was empty. After `export SHELL=/bin/bash ` everything works OK

**Comment by wagoodman** [comment]
Nice find! I've updated master with the fix f0bd02f93a09c5f0e1ee55aebb384efeb9a93849, at least bashful will default to `sh` in cases where the SHELL env var is not configured.
